src/ansys/additive/core/single_bead.py

```diff
@@ -120,12 +137,38 @@ def bead_length(self, value):
         self.__validate_range(value, self.MIN_BEAD_LENGTH, self.MAX_BEAD_LENGTH, "bead_length")
         self._bead_length = value
 
+    @property
+    def output_thermal_history(self) -> bool:
+        """Flag to output the thermal history of the simulation."""
```

Comment:
```suggestion
        """Option to output the thermal history of the simulation."""
```

Reply:
For a property, the format is generally: "Flag indicating whether to output the thermal history of the simulation."

Reply:
OK good, go with that! Thanks, Kathy.